src/main/java/digital/slovensko/autogram/core/eforms/FsEFormResources.java

```diff
@@ -0,0 +1,135 @@
+package digital.slovensko.autogram.core.eforms;
+
+import digital.slovensko.autogram.core.eforms.dto.EFormAttributes;
+import digital.slovensko.autogram.core.errors.EFormException;
+import digital.slovensko.autogram.core.errors.UnknownEformException;
+import digital.slovensko.autogram.core.errors.XMLValidationException;
+import eu.europa.esig.dss.enumerations.DigestAlgorithm;
+import eu.europa.esig.dss.model.InMemoryDocument;
+
+import static digital.slovensko.autogram.core.eforms.EFormUtils.*;
+
+public class FsEFormResources extends EFormResources {
+    private static final String SOURCE_URL = "https://forms-slovensko-digital.s3.eu-central-1.amazonaws.com/fs/";
+    private String xdcIdentifier;
+
+    public FsEFormResources(String fsFormId, String canonicalizationMethod, String xsdDigest, String xsltDigest) {
+        super(fsFormId, xsdDigest, xsltDigest, canonicalizationMethod);
+        this.embedUsedSchemas = false;
+    }
+
+    public static FsEFormResources buildFromXdcIdentifier(String xdcIdentifier, String canonicalizationMethod, String xsdDigest, String xsltDigest) {
+            return new FsEFormResources(getFormIdFromXdcIdenitfier(xdcIdentifier), canonicalizationMethod, xsdDigest, xsltDigest);
+    }
+
+    private static String getFormIdFromXdcIdenitfier(String xdcIdentifier) {
+        xdcIdentifier = xdcIdentifier.replaceAll("[:./ ]", "_");
+        var forms_xml = getResource(SOURCE_URL + "forms.xml");
+        if (forms_xml == null)
+            throw new XMLValidationException("Zlyhala príprava elektronického formulára", "Nepodarilo sa nájsť zoznam FS formulárov");
+
+        var parsed_meta_xml = getXmlFromDocument(new InMemoryDocument(forms_xml, "forms.xml"));
+        var nodes = parsed_meta_xml.getElementsByTagName("sd:" + xdcIdentifier);
```

Comment:
Tento namespace alias natvrdo ma tu trosku vyrusuje. Co ak niekedy pouziju iny NS? Nepouzijeme radsej priamo `getElementsByTagNameNS`?

Reply:
ten sd je naša custom vec (sd ~ Slovensko Digital). Ale hej, môžem to tam dať cez namespace, ten máme zadefinovaný.